### PR TITLE
 delete watermarking

### PR DESCRIPTION
**Pull request by HaoXuan40404**

### CODE REVIEW

**Comment by HaoXuan40404** [comment]
overview at https://haoxuan.readthedocs.io/en/feature-2.0.0/docs/faq.html

File: docs/faq.md [hunk 1]
```diff
@@ -1,123 +1,150 @@
...
+答:
   节点加入记账列表和观察者列表的节点必须是连接peer的nodeID列表里面的成员。
 
-:问:
-  删除节点操作报错，nodeID is not in group peers，为什么？
-:答:
+问:
+  删除节点操作报错，nodeID is not in group peers，为什么？  
+答:
   节点删除操作中的节点必须是getGroupPeers里面展示的group的peers。
```
Comment (chaychen2005) [inline docs/faq.md]: Description is not rigorous. Do not require the node has been connected.

File: docs/faq.md [hunk 2]
```diff
@@ -1,123 +1,150 @@
...
+  FISCO BCOS v2.0如何搭建？<br>
+答:
+  FISCO BCOS支持多种搭建方式，常用方式有：
+  - 建链脚本：使用简单，适合第一次接触FISCO BCOS的同学，具体请 [参考这里](./manual/build_chain.md)。
+  - FISCO-Generator：更灵活，适合熟悉FISCO BCOS的用户和企业用户，具体请 [参考这里](./enterprise/index.html)。
+
+问:
+  FISCO BCOS v2.0的智能合约与之前版本合约有什么不同，兼容性如何？<br>
+答:
+  FISCO BCOS v2.0支持最新的Solidity合约，同时增加了precompare合约，具体请 [参考这里](developer/index.html)。
```
Comment (cyjseagull) [inline docs/faq.md]: precompare->precompile

File: docs/faq.md [hunk 3]
```diff
@@ -1,123 +1,150 @@
...
+
+问:
+  国密和普通版本的区别有哪些？<br>
+答:
+  编译版本，证书，落盘加密，solidity编译java，web3sdk使用，具体请 [参考这里](./manual/guomi.md)。
```
Comment (cyjseagull) [inline docs/faq.md]: Description is a little simple. Maybe need link to a more detailed reference.

File: docs/faq.md [hunk 4]
```diff
@@ -1,123 +1,150 @@
...
-""""""""""""""""
+## FISCO BCOS使用
 
-:问:
-  系统配置、群组配置、节点配置分别指什么？
-:答:
+问:
+  系统配置、群组配置、节点配置分别指什么？<br>
+答:
   系统配置是指节点配置中一些影响账本功能，并需账本节点共识的配置项。群组配置指节点所属的群组的相关配置，节点的每个群组都有独立的配置。节点配置指所有可配置项。
```
Comment (cyjseagull) [inline docs/faq.md]: Maybe should give examples of group configuration(group.${group_id}.genesis and group.${group_id}.ini), node configuration(config.ini)

File: docs/faq.md [hunk 5]
```diff
@@ -1,123 +1,150 @@
...
-  - 手工可改配置：group.x.ini文件中，修改后重启生效，只影响节点。配置项有[tx_pool].limit。
+  - 手工可改配置：`group.x.ini`文件中，修改后重启生效，只影响节点。配置项有[tx_pool].limit。
 
-:问:
-  群组共识可改配置如何更改、查询？
-:答: 共识可改配置可以通过控制台修改。共识可改配置项查询除了控制台外，还可以通过RPC接口查询，具体请 :doc:`参考这里 <api/rpc>`。
+问:
+  群组共识可改配置如何更改、查询？<br>
+答: 
+  共识可改配置可以通过控制台修改。共识可改配置项查询除了控制台外，还可以通过RPC接口查询，具体请 [参考这里]api/rpc。
```
Comment (cyjseagull) [inline docs/faq.md]: This seems has some problem, add the brackets outside `api/rpc.md`

File: docs/faq.md [hunk 6]
```diff
@@ -1,123 +1,150 @@
...
   + [consensus].node.X的更改涉及到节点管理，控制台接口涉及到addMiner(am)，addObserver(ao)，removeNode(rn)，具体参考《节点管理》。
 
-:问:
-  群组观察节点和共识节点有什么区别？
-:答:
+
+问:
+  群组观察节点和共识节点有什么区别？<br>
+答:
   观察节点能同步群组数据，但不能参与共识。共识节点除了具有观察者权限，还参与共识。
```
Comment (cyjseagull) [inline docs/faq.md]: Maybe describe the function of observer node and consensus node is better. eg:

observer node: sync blocks and transactions;
consensus node: sync blocks and transactions, take part in consensus procedure.

File: docs/faq.md [hunk 7]
```diff
@@ -1,123 +1,150 @@
...
 
-:问:
-  如何将合约纳入CNS管理？
-:答:
+
+问:
+  如何将合约纳入CNS管理？<br>
+答:
```
Comment (cyjseagull) [inline docs/faq.md]: url of CNS operation guide is essential.

File: docs/faq.md [hunk 8]
```diff
@@ -1,123 +1,150 @@
...
   在部署合约时，调用CNS合约接口，将合约name、version、address信息写入CNS表中
 
-:问:
-  如何查询合约CNS表？
-:答:
+
+问:
+  如何查询合约CNS表？<br>
+答:
```
Comment (cyjseagull) [inline docs/faq.md]: reference url is essential.

File: docs/faq.md [hunk 9]
```diff
@@ -1,123 +1,150 @@
...
   推荐使用oracle jdk 1.8（open-jdk 1.8在某些操作系统上会缺少椭圆曲线的包。） 
 
-:问:
-  Java SDK配置完成，发送交易失败的原因是什么？  
-:答:
+
+问:
+  Java SDK配置完成，发送交易失败的原因是什么？<br>
+答:
   applicationContext.xml中的ip、端口、群组号填错或者是缺少节点的ca.crt、node.crt和node.key文件。
```
Comment (cyjseagull) [inline docs/faq.md]: Another reason:

channel ip of node is 127.0.0.1, and the sdk is deployed in another machine.

maybe should describe more detailed in sdk document, and link to the sdk faq.

File: docs/faq.md (hunk 2 above)
Comment (HaoXuan40404) [reply docs/faq.md]: done

File: docs/faq.md (hunk 3 above)
Comment (HaoXuan40404) [reply docs/faq.md]: done

File: docs/faq.md (hunk 5 above)
Comment (HaoXuan40404) [reply docs/faq.md]: done

**Pull request merged**